modules/security-context-constraints-psa-rectifying.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+:_content-type: PROCEDURE
+[id="security-context-constraints-psa-rectifying_{context}"]
+= About pod security admission alerts 
+
+A `PodSecurityViolation` alert is triggered when the Kubernetes API server reports that there is a pod denial on the audit level of the pod security admission controller.
+
+View the Kubernetes API server (kube-apiserver) audit logs to investigate alerts that were triggered. As an example, a workload is likely to fail admission if global enforcement is set to the `restricted` pod security level.
```

Comment:
```suggestion
View the Kubernetes API server (kube-apiserver) audit logs to investigate alerts that were triggered. As an example, a workload is likely to fail admission if global enforcement is set to the `restricted` pod security level. Therefore, this alert can help you check what are the workloads which have not been admitted. 
```

I think we need to better clarify here:
a) The purpose of the alert ( it does not help us to know the workload which if / when we change the enforcement to restricted would fail. Am I right? 
b) When the alert is fired ( only after 1 day ) 
c) for how long the alert is there. When the alert goes away?

Reply:
What you're proposing is incorrect, that's not how the audit annotations works.

Which brings me on to the docs itself - the annotation will be set both:
- when the admission fails
- when the admission would fail if the enforcement were set to the level the audit label requires it

Back to the questions from the comment:
a) Yes, and the docs state that in order to be able to see which pod triggered the violation, you need to check the audit logs
b) false, the alert is fired right away
c) indeed, we shoud document it (it's one day)

Reply:
Add it to line 9: `This alert is persisted for one day.` (it's not in the logs, btw, it's an alert)